tests/test_api.py

```diff
@@ -50,7 +50,10 @@
 COUNTRY_CODE_INVALID_ERROR = "country code must be a 2 length alpha string - %(code)s given"
 ENROLLMENT_PERIOD_INVALID_ERROR = "cannot be greater than "
 ENROLLMENT_PERIOD_OUT_OF_BOUNDS_ERROR = "%(type)s %(date)s is out of bounds"
-NOT_FOUND_ERROR =  "%(entity)s not found in the registry"
+NOT_FOUND_ERROR = "%(entity)s not found in the registry"
+GENDER_ACC_INVALID_ERROR = "gender accuracy is set only when gender is given"
+GENDER_ACC_INVALID_TYPE_ERROR = "gender_acc must have an integer value"
+GENDER_ACC_INVALID_RANGE_ERROR = "gender accuracy is not in range (0,100"
```

Comment:
(0,100) ?

Reply:
You're probably right and 0 doesn't make any sense.